src/audio/channel3d.js

```diff
@@ -162,10 +170,6 @@ if (!hasAudioContext()) {
             }
         },
 
-        setVelocity: function (velocity) {
-            this.velocity.copy(velocity);
-        },
-

```

Comment:
To confirm, by removing this line, will the default implementation (on lines 70-72) be used?

Reply:
Correct! 😄